examples/vacationsApprovalAzureFunctions/README.md

```diff
@@ -0,0 +1,105 @@
+
+Next, you need to install dependencies:
+```
+npm install
+```
+
+#### Test the Function locally
+
+Run the following command to run the function locally:
+```
+func start
+```
+You will see the url to test the function.
+Now you can test it with Postman or from a browser.
+
+#### Deploy the Azure Function
+
+In order to deploy the function to Azure, first you need to create a resorce group and specify the location:
+```
+az group create --name CalculateDaysOff-rg --location eastus
+```
+
+Next, you need to create a general-purpose storage account for the resource group and region:
+```
+az storage account create --name <STORAGE_NAME> --location westeurope --resource-group AzureFunctionsQuickstart-rg --sku Standard_LRS
+```
+Then, create the funciton app, in the following command, replace <STORAGE_NAME> with the name of the account you used in the previous step, and replace <APP_NAME> with a globally unique name appropriate to you. The <APP_NAME> is also the default DNS domain for the function app.
+```
+az functionapp create --resource-group CalculateDaysOff-rg --consumption-plan-location eastus --runtime node --runtime-version 10 --functions-version 2 --name <APP_NAME> --storage-account <STORAGE_NAME>
+```
+
+The last step is deploy the function:
+```
+func azure functionapp publish <APP_NAME>
+```
+When the process is done, you will see the `Invoke url`, the url include a `code` query string. You should keep this code so you can use it with the smartmate process.
+
+## Setup Smartmate Process
+
+Before starting, please take a look at the [Getting Started](https://docs.apps.smartmate.io/getting-started/index.html "Smartmate Getting Started Guide") page to get familiar with the `smartmate-cli`.
+
+Once you have your workspace set up, you can add this app by running: `sm add app` from inside your workspace, and selecting this app. Now you can go to the `processes/requestVacations.yml` file.
+
+You will see the `serviceTasks` property with a `calculateDaysOff` service tasks.
+Here in the `url` property you can paste the endpoint deployed to GCloud.
+```
+serviceTasks:
+  calculateDaysOff:
+    name: Calculate Days Off
+    type: httpRequest
+    retries: 3
+    method: GET
+    url: <PLACE YOUR ENDPOINT URL>
+    query:
+      code: "{{_secrets.azureCode}}"
+      startDate: "{{startDate}}" 
+      endDate: "{{endDate}}"
+    resultMapping:
+      body.data: daysOff
+```
+This service task will take the values of `startDate` and `endDate` fields and send it to the endpoint like query strings. Also you need to set the `azureCode` secret to be used in the process.
+The response of the call will fill the `daysOff` field so it can be visibly in the process.
+
+Now you can publish your app.
+```
+sm publish -f
+```
+Before test the Smarmate process you need to set the secret `azureCode` in the app settings in the smartmate web app.
```

Comment:
Where are the app settings? Probably you can add a little more description. (or eventually we can add a link to a page in the smartmate documentation on how to add secrets)